packages/video_player/video_player_avfoundation/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: video_player_avfoundation
 description: iOS implementation of the video_player plugin.
 repository: https://github.com/flutter/plugins/tree/master/packages/video_player/video_player_avfoundation
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+video_player%22
-version: 2.2.17
+version: 2.2.18
```

Comment:
@stuartmorgan is there anything else I need to do to version this federated plugin?

Reply:
No, we don't push version changes through to the app-facing plugin; the expectation is that people are updating transitive dependencies.